benchmarks/query_benchmarks/query_delete/benchmark.py

```diff
@@ -0,0 +1,15 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryDelete:
+    def setup(self):
+        bench_setup(migrate=True)
+        for i in range(0, 10):
+            Book(pk=i, title=f"foobar_{i}").save()
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_delete(self):
+        Book.objects.all().delete()
```

Comment:
https://github.com/smithdc1/django-asv/runs/6720503551?check_suite_focus=true#step:5:61

This benchmark is super quick too. I wonder if we could delete each item in turn? Not sure if that gives a materially different flow on Django's side to delete all.

Reply:
To delete each item in turn I used the following method

```python
class QueryDelete:
    def setup(self):
        bench_setup(migrate=True)
        self.objects = Book.objects.all()

    def time_query_delete(self):
        for obj in self.objects:
            obj.delete()
```
but I am getting the following error `ValueError: Book object can't be deleted because its id attribute is set to None.`. Putting the self.objects statement in `time_query_delete` works fine

Reply:
ok, let's not worry about it for now.